content/api/commands/mount.md

```diff
@@ -127,7 +132,7 @@ example as a starting point for customizing your own command:
 <template #react>
 
 ```ts
-import { MountOptions, MountReturn } from '@cypress/react'
+import { MountOptions, MountReturn } from 'cypress/react'
```

Comment:
Do we know that `MountOptions` and `MountReturn` will still be importable in the same way from `cypress/react`?

Reply:
I asked the team to verify

Reply:
They verified that all exports should be the same in the new bundled package.